build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildSetTask.java

```diff
@@ -126,7 +126,22 @@ public void addBuildTask(BuildTask buildTask) {
         buildTasks.add(buildTask);
     }
 
-    public Integer getId() {
+    /**
+     * Get the build task which contains the given build configuration
+     * 
+     * @param buildConfig
+     * @return The build task with the matching configuration, or null if there is none
+     */
+    public BuildTask getBuildTask(BuildConfiguration buildConfig) {
```

Comment:
I find it ok being here. Looks like it can be converted in stylish J8 one-liner ;) .strem.filter.findFirst ;)